app/controllers/location_shifts_controller.rb

```diff
@@ -46,13 +46,13 @@ def destroy
 
     @location_shift.destroy
 
-    redirect_to location_shift_path(location_id: @location_shift.location_uid)
+    redirect_to location_shift_path(organisation_id: @location_shift.organisation_id)
   end
 
   private
 
-  def location
-    find_organisation_by_uid(uid: params[:location_id])
+  def organisation
+    Organisation.find(params[:organisation_id])
   end
 
   def location_shift
```

Comment:
Maybe we should rename this method and also the `location_shift_params_from` to be inline with the `organisation` naming.